Controller/PositionableController.php

```diff
@@ -58,43 +65,48 @@ public function decreasePositionAction(CRUDElement $element, $id, Request $reque
     }
 
     /**
-     * @param \FSi\Bundle\AdminBundle\Doctrine\Admin\CRUDElement $element
+     * @param DataIndexerElement $element
      * @param int $id
-     * @throws \RuntimeException
+     * @throws RuntimeException
      * @return PositionableInterface
      */
-    private function getEntity(CRUDElement $element, $id)
+    private function getEntity(DataIndexerElement $element, $id)
     {
         $entity = $element->getDataIndexer()->getData($id);
 
         if (!$entity instanceof PositionableInterface) {
-            throw new \RuntimeException(sprintf('Entity with id %s does not implement PositionableInterface', $id));
+            throw new RuntimeException(
+                sprintf('Entity with id %s does not implement PositionableInterface', $id)
+            );
         }
 
         return $entity;
     }
 
     /**
-     * @param CRUDElement $element
+     * @param DataIndexerElement $element
      * @param Request $request
      * @return RedirectResponse
      */
-    private function getRedirectResponse(CRUDElement $element, Request $request)
+    private function getRedirectResponse(DataIndexerElement $element, Request $request)
     {
         if ($request->query->get('redirect_uri')) {
             $uri = $request->query->get('redirect_uri');
         } else {
-            $uri = $this->router->generate($element->getRoute(), $element->getRouteParameters());
+            $uri = $this->router->generate(
+                $element->getRoute(),
+                $element->getRouteParameters()
+            );
         }
 
         return new RedirectResponse($uri);
     }
 
     /**
-     * @param CRUDElement $element
+     * @param DataIndexerElement $element
      * @param $entity
      */
-    private function persistAndFlush(CRUDElement $element, $entity)
+    private function persistAndFlush(DataIndexerElement $element, $entity)
```

Comment:
this type hint could be changed to ``FSi\Bundle\AdminBundle\Doctrine\Admin\Element`` to ensure it has ``getObjectManager()`` method